aten/src/ATen/DLConvertor.cpp

```diff
@@ -69,29 +69,29 @@ DLDataType getDLDataType(const Tensor& t) {
     case ScalarType::Float8_e4m3fn:
     case ScalarType::Float8_e4m3fnuz:
     case ScalarType::Float8_e8m0fnu:
-      TORCH_CHECK(false, "float8 types are not supported by dlpack");
+      TORCH_CHECK_BUFFER(false, "float8 types are not supported by dlpack");
```

Comment:
maybe the more specific error would be not supported by 1.0, it is supported for example by 1.1 now https://github.com/dmlc/dlpack/releases

Reply:
Sure we could do that. But, maybe in a later PR.